src/RadioButton/index.js

```diff
@@ -0,0 +1,121 @@
+// @flow
+
+import * as React from "react";
+import { darken } from "polished";
+import * as tokens from "orbit-design-token";
+
+type Props = {
+  name: string,
+  label: string,
+  value: string,
+  error?: boolean,
+  disabled: boolean,
```

Comment:
disabled is requred, but checked is defined as optional and both have defaults => I would suggest to unify this and define them as required with defaults

Reply:
I'm going to define all props as required as you suggested.